### PR TITLE
template building

### PR DESCRIPTION
**Pull request by davidakabogu**
initiate team page, testing branches, pull request and pair programming

### CODE REVIEW

**Review by Rashotech — COMMENTED** [review]
Well done, this all? I would suggest should store the team members data in an array of object and map through it. the Team member card too should be in a separate components

File: src/pages/ProjectMembers.jsx [hunk 1]
```diff
@@ -1,9 +1,56 @@
...
-}
+    // team members page
+    <div className="flex items-center bg-white">
+      <div>
+        <p className="font-bold text-blue-900 text-4xl mt-8 ">
+          Meet the Team...
+        </p>
+        <div className="flex flex-wrap mt-8 mx-16 font-bold">
+          {teamMembers.map((member) => (
+            <div
```
Comment (Rashotech) [inline src/pages/ProjectMembers.jsx]: can you extract the Team Card as a component and and put it in the components folder. Then import it here and passing the member data as props to it . 

`<TeamCard team={team} /> `

File: src/pages/ProjectMembers.jsx [hunk 2]
```diff
@@ -1,9 +1,56 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import { getProjectMembers } from "../services/teams";
 
 const ProjectMembers = () => {
+  const [teamMembers, setTeamMembers] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
```
Comment (Rashotech) [inline src/pages/ProjectMembers.jsx:11]: I fetched data from the services here

**Comment by davidakabogu** [comment]
kindly let me know if its been done the way you wish and if there are still improvements to be made

**Pull request merged**